configstack/stack.go

```diff
@@ -34,19 +33,31 @@ func (stack *Stack) Graph(terragruntOptions *options.TerragruntOptions) {
 	WriteDot(terragruntOptions.Writer, terragruntOptions, stack.Modules)
 }
 
-// Plan execute plan in the given stack in their specified order.
-func (stack *Stack) Plan(terragruntOptions *options.TerragruntOptions) error {
-	stack.setTerraformCommand([]string{"plan"})
+func (stack *Stack) Run(terragruntOptions *options.TerragruntOptions) error {
+	stackCmd := terragruntOptions.TerraformCommand
+	// For apply and destroy, run in non-interactive mode to avoid comingling stdin across multiple concurrent runs.
```

Comment:
Why only apply and destroy? Couldn't there be `stdin` for, say, `plan`? E.g., If you're missing a variable?

Reply:
Ah yup that is true. Fixed to use the list of commands tracked in `config/config_helpers.go`: 509598898b8b5897367a11b2a5569877c91e0934